pkg/apis/bugs/v1/types.go

```diff
@@ -0,0 +1,22 @@
+package v1
+
+import "time"
+
+// Bug is used to represent bugs in some serialized content.  It also appears to be used internally for data tracking
+type Bug struct {
+	BugzillaBug  `json:",inline"`
+	Url          string `json:"url"`
+	FailureCount int    `json:"failureCount,omitempty"`
+	FlakeCount   int    `json:"flakeCount,omitempty"`
+}
+
+// BugzillaBug matches the bugzilla API.  We cannot change this and should consider writing a converter instead of having
+// a serialization we don't own.
+type BugzillaBug struct {
+	ID             int64     `json:"id"`
+	Status         string    `json:"status"`
+	LastChangeTime time.Time `json:"last_change_time"`
+	Summary        string    `json:"summary"`
+	TargetRelease  []string  `json:"target_release"`
```

Comment:
wonder why is this an []string, can you have more than 1 target release?

Reply:
yes.  the BZ api returns it as an array.  though i haven't figured out how set more than one in the UI.